utensor_cgen/backend/utensor/_backend_impl.py

```diff
@@ -31,7 +32,7 @@ def __init__(
     if config['legacy-api']:
       code_generator = code_generator or uTensorLegacyCodeGenerator(config=config[uTensorLegacyCodeGenerator.PART].to_dict())
       graph_op_lower = graph_op_lower or uTensorLegacyGraphLower(config=config[uTensorLegacyGraphLower.PART].to_dict())
-      graph_alloc_lower = graph_alloc_lower or TensorAllocationPlanner(config=config[TensorAllocationPlanner.PART].to_dict())
+      graph_alloc_lower = graph_alloc_lower or BrutalForceMemoryPlanner(config=config[BrutalForceMemoryPlanner.PART].to_dict())
```

Comment:
I think considering the performance of brutal force mem planner, it's better to make TensorAllocationPlanner as default planner.